Meadowlark-js/packages/meadowlark-authz-server/package.json

```diff
@@ -13,6 +13,7 @@
     "/package.json"
   ],
   "dependencies": {
+    "@apideck/better-ajv-errors": "^0.3.6",
```

Comment:
Trying out this ajv error writer. Joi's default error messages were an order of magnitude better. Ajv's are more meant for custom error message assembly. If we like it, we can roll into meadowlark-core.